README.md

```diff
@@ -53,9 +53,10 @@ value ease of use and speed of learning over architectural flexibility. You can
 join Backdrop's developer community by
 [forking Backdrop](https://github.com/backdrop/backdrop) on GitHub.
 
-When submitting a pull request, please make sure there is an open issue in the
-[issue tracker](https://github.com/backdrop/backdrop-issues/issues) and
-reference it in the description on the pull request.
+You can submit changes to Backdrop by creating a pull request that references an
+issue in the [issue tracker](https://github.com/backdrop/backdrop-issues/issues).
+Details about how to do this are in the [Contribute to Backdrop Core](https://docs.backdropcms.org/documentation/contribute-to-backdrop
+-core) documentation.
```

Comment:
Looks like GitHub's markdown parser doesn't handle new-lines in URLs.

![image](https://user-images.githubusercontent.com/100206/166163036-56286b58-86d0-4f84-bd03-03c50a52d579.png)

I think this is the closest we can get to staying within an 80 character line wrap:

```suggestion
Details about how to do this are in the
[Contribute to Backdrop Core](https://docs.backdropcms.org/documentation/contribute-to-backdrop-core)
documentation.
```